adapters/adapters-base/src/main/java/org/apache/cassandra/sidecar/adapters/base/jmx/StorageJmxOperations.java

```diff
@@ -233,4 +233,11 @@ public interface StorageJmxOperations
      * @return the current compaction throughput in megabytes per second, or 0 if throughput cannot be determined
      */
     int getCompactionThroughputMbPerSec();
+
+    /**
+     * Triggers the node move operation to move this node to a new token
+     *
+     * @param newToken the new token for the node to move to
+     */
+    void move(String newToken);
```

Comment:
We need to propagate any potential IOException from JMX and we should handle it 
```suggestion
    void move(String newToken) throws IOException;
```

Reply:
Updated the method signature. OperationalJob.execute() already handles the exception. I also added tests for node move failure.